test/restart-containers.sh

```diff
@@ -6,4 +6,4 @@ docker-compose up -d mongo
 docker-compose rm -sf ganache-cli
 docker-compose up -d ganache-cli
 docker-compose restart listener
-cd dex-contracts && wait-port -t 30000 8545 && truffle migrate && cd -
\ No newline at end of file
+cd dex-contracts && npx wait-port -t 30000 8545 && npx truffle migrate && cd -
```

Comment:
what is wait-port? Is this a truffle command?

Reply:
wait-port is something that allows you to wait for one service to be up.
It was added by Felix to coordinate better when the migration is executed.

I just use npx so we don't forze users to have it installed in their computer:
https://www.npmjs.com/package/wait-port